packages/expo-dev-menu/ios/EXDevMenuAppInfo.m

```diff
@@ -22,24 +23,30 @@ @implementation EXDevMenuAppInfo
 //- (nullable NSString *)facebookApplicationName;
 //- (BOOL)facebookAutoInitEnabled;
 
-+(NSDictionary *)getAppInfoForBridge:(RCTBridge *)bridge andManifest:(NSDictionary *)manifest
++(NSDictionary *)getAppInfo
 {
   NSMutableDictionary *appInfo = [NSMutableDictionary new];
-
+  
   NSString *appIcon = [EXDevMenuAppInfo getAppIcon];
   NSString *runtimeVersion = [EXDevMenuAppInfo getUpdatesConfigForKey:@"EXUpdatesRuntimeVersion"];
   NSString *sdkVersion = [EXDevMenuAppInfo getUpdatesConfigForKey:@"EXUpdatesSDKVersion"];
   NSString *appVersion = [EXDevMenuAppInfo getFormattedAppVersion];
   NSString *appName = [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleDisplayName"] ?: [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleExecutable"];
-  NSString *hostUrl = [bridge.bundleURL host] ?: @"";
-
-  if (manifest[@"name"] != nil) {
-    appName = manifest[@"name"];
-  }
   
-  if (manifest[@"version"] != nil) {
-    appVersion = manifest[@"version"];
+  DevMenuManager *manager = [DevMenuManager shared];
```

Comment:
keeping the manifest as a parameter (and having the caller be the one to pass in `DevMenuManager.shared.currentManifest`) might help keep this module more easily testable

Reply:
hmm yea that's true - I would prefer to mock the devmenumanager singleton if possible and keep all this logic related to getting app info fields all in one place